src/main/java/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,56 @@
+package lotto;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    private final List<LottoNumber> numbers;
+
+    public LottoNumbers(List<LottoNumber> numbers) {
+        this.numbers = numbers;
+    }
+
+    public void shuffleNumbers() {
+        Collections.shuffle(numbers);
+    }
+
+    public LottoNumbers selectNumbers(int count) {
+        return new LottoNumbers(numbers.subList(0, count)
+                .stream()
+                .sorted()
+                .collect(Collectors.toList()));
+    }
+
+    public int winningCount(LottoNumbers winningNumbers) {
+        int count = 0;
+
+        for (LottoNumber lottoNumber: winningNumbers.numbers) {
+            if (this.numbers.contains(lottoNumber)) {
+                count++;
+            }
+        }
```

Comment:
if 문을 별도의 메소드로 분리해 보면 어떨까요?
프로그래밍 요구사항 중 indent 관련 요구사항을 참고해 보시면 좋을 것 같아요.

컨벤션도 확인해 주세요.
```suggestion
        for (LottoNumber lottoNumber : winningNumbers.numbers) {
            if (this.numbers.contains(lottoNumber)) {
                count++;
            }
        }
```